README.md

```diff
@@ -19,14 +19,24 @@ So, the all-new goreleaser was born.
 
 ## Usage
 
-Basically, you need to create a `goreleaser.yml` file in the root of your
+```
+go get github.com/goreleaser/releaser
```

Comment:
Both work. To develop against the project/ get stuff not yet in a release then `go get` works. To use the latest, supported version to, say, test a new `goreleaser.yml` file then development or not: you should be using the newest release.

Certainly the latest released version is the supported method of pushing stuff to github.